src/modules/items/ItemList.ts

```diff
@@ -201,3 +200,6 @@ ItemList.Carbos   = new Vitamin(VitaminType.Carbos, 1e5, Currency.money, {
     saveName: `${VitaminType[VitaminType.Carbos]}|${Currency[Currency.money]}`,
     visible: new MaxRegionRequirement(Region.unova),
 }, undefined, 'Reduces steps required when hatching');
+
+// Consumables
+ItemList.RareCandy = new Consumable(ConsumableType.RareCandy, Infinity, undefined, undefined, 'Rare Candy', 'A rare-to-find Candy<br/>Keep away from children!<br/>Increases the attack of a Pokémon as if bred');
```

Comment:
`Rare_Candy` instead of `RareCandy` would follow the style of the rest of the item names